### PR TITLE
updated ECR information

### PR DESCRIPTION
**Pull request by Autumn808**
Updating information on ECR.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,25 +8,24 @@ The docker images are available on [Amazon Corretto Official Images](https://hub
 
 To use Amazon Corretto Official Images from Docker hub run
```
Comment (lutkerd) [inline README.md:9]: Include link to [dockerhub offical image ](https://hub.docker.com/_/amazoncorretto)

File: README.md [hunk 2]
```diff
@@ -8,25 +8,24 @@ The docker images are available on [Amazon Corretto Official Images](https://hub
...
-docker pull amazoncorretto:11
+docker pull amazoncorretto:latest
 ```
 
 The docker images are also available on **Amazon ECR**.
 
-To use the docker images from Amazon ECR, you would need to authenticate with the ECR registry (id: 489478819445) with the
-help of instruction from [here](https://aws.amazon.com/blogs/compute/authenticating-amazon-ecr-repositories-for-docker-cli-with-credential-helper/).
-Once authenticated, Amazon Corretto docker images can be pulled using command
+To use the docker images from Amazon ECR, avilable images can be found in the public ECR repository.
```
Comment (lutkerd) [inline README.md:16]: This doesn't quite flow right to me.. Maybe something like this:

To get Corretto docker images from Amazon ECR please see [Amazon Corretto's ECR Public Gallery](https://gallery.ecr.aws/amazoncorretto/amazoncorretto) as well as the [Docker Official Images ECR Public Gallery](https://gallery.ecr.aws/docker/library/amazoncorretto) 


typo: available

File: README.md [hunk 3]
```diff
@@ -8,25 +8,24 @@ The docker images are available on [Amazon Corretto Official Images](https://hub
...
-To use the docker images from Amazon ECR, you would need to authenticate with the ECR registry (id: 489478819445) with the
-help of instruction from [here](https://aws.amazon.com/blogs/compute/authenticating-amazon-ecr-repositories-for-docker-cli-with-credential-helper/).
-Once authenticated, Amazon Corretto docker images can be pulled using command
+To use the docker images from Amazon ECR, avilable images can be found in the public ECR repository. 
 
-```
-docker pull 489478819445.dkr.ecr.us-west-2.amazonaws.com/amazoncorretto:latest
-docker run -it 489478819445.dkr.ecr.us-west-2.amazonaws.com/amazoncorretto:latest /bin/bash
-```
+To use the Corretto ECR instance of the following commands:
```
Comment (lutkerd) [inline README.md:18]: To use docker images from Corretto ECR Public Gallery run the following commands:

Comment (Autumn808) [reply README.md]: Thank you for taking the time to look at my PR I added the fixes for your comments.

File: README.md (hunk 1 above)
Comment (Autumn808) [reply README.md:9]: The link is in line 7

File: README.md [hunk 4]
```diff
@@ -8,25 +8,24 @@ The docker images are available on [Amazon Corretto Official Images](https://hub
...
-docker pull 489478819445.dkr.ecr.us-west-2.amazonaws.com/amazoncorretto:latest
-docker run -it 489478819445.dkr.ecr.us-west-2.amazonaws.com/amazoncorretto:latest /bin/bash
-```
+To use docker images from Corretto ECR Public Gallery run the following commands:
 
-You can view the available tags, run
 ```
-aws ecr list-images --region us-west-2 --registry-id 489478819445 --repository-name amazoncorretto | jq -r '.imageIds[] | .imageTag'
+docker pull public.ecr.aws/amazoncorretto/amazoncorretto:latest
+docker run -it public.ecr.aws/amazoncorretto/amazoncorretto:latest /bin/bash/amazoncorretto:latest /bin/bash
```
Comment (Rudometov) [inline README.md:22]: This command wont work

**Pull request closed without merging**